app/views/moirai/translation_files/show.html.erb

```diff
@@ -30,8 +30,7 @@
                          method: :post do |f| %>
               <%= f.hidden_field :key %>
               <%= f.hidden_field :locale %>
-              <%= f.text_area :value, class: 'translation-textarea' %>
-              <%= f.submit 'Update', style: 'display: none;' %>
+              <%= f.text_field :value, class: 'translation-textarea' %>
```

Comment:
Use text field because you cannot submit inside of a textarea. Also the submit button is useless.